### PR TITLE
DestVI User Guide

### PR DESCRIPTION
**Pull request by jjhong922**

### CODE REVIEW

File: docs/user_guide/models/destvi.rst [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+To avoid overfitting, DestVI amortizes inference using a neural network to parametrize the latent variables.
+Via the ``amortization`` parameter of :class:`scvi.module.MRDeconv`, the user can specify which of
+:math:`\beta` and :math:`\gamma^c` will be parametrized by the neural network.
+
+
+Tasks
+=====
+
+Cell type deconvolution
+-------------
```
Comment (adamgayoso) [inline docs/user_guide/models/destvi.rst:161]:
```suggestion
Cell type deconvolution
-------------------------
```

File: docs/user_guide/models/destvi.rst [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+Once the model is trained, one get retrieve the estimated cell type proportions in each spot using the method::
```
Comment (adamgayoso) [inline docs/user_guide/models/destvi.rst:162]:
```suggestion
Once the model is trained, one can retrieve the estimated cell type proportions in each spot using the method::
```

**Review by adamgayoso — CHANGES_REQUESTED** [review]

File: docs/user_guide/models/destvi.rst [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+   * - :math:`\gamma_s^c \in `
+     - Inverse dispersion for negative binomial. This can be set to be gene/batch specific for example (and would thus be :math:`\theta_{kg}`), by passing ``dispersion="gene-batch"`` during model intialization. Note that ``px_r`` also refers to the underlying real-valued torch parameter that is then exponentiated on every forward pass of the model.
+     - ``gamma``
+   * - :math:`p_g \in \mathbb{R}^{+}`
+     - Inverse dispersion for negative binomial. This can be set to be gene/batch specific for example (and would thus be :math:`\theta_{kg}`), by passing ``dispersion="gene-batch"`` during model intialization. Note that ``px_r`` also refers to the underlying real-valued torch parameter that is then exponentiated on every forward pass of the model.
+     - ``px_o``
+
+
+Inference
+=========
```
Comment (adamgayoso) [inline docs/user_guide/models/destvi.rst:133]: I'd break this out by sc and st models

Comment (adamgayoso) [reply docs/user_guide/models/destvi.rst:133]: Also should be q(gamma)?

Comment (adamgayoso) [reply docs/user_guide/models/destvi.rst:133]: And finally for scLVM I'd refer to the VI tutorial, for stLVM I'd write out the loss explicitly

File: docs/user_guide/models/destvi.rst [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+for spatial data, we assume that each spot :math:`s` has expression :math:`x_s` composed of gene expression
+counts of :math:`C(s)` individual cells. We assume that cell :math:`n` in spot :math:`s` is generated from
+the latent variables :math:`(c_{ns}, \gamma_{ns})` by the following process:
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+        x_{sg} &\sim \mathrm{NegativeBinomial}(l_s\alpha_g\sum_{n=1}^{C(s)}f^g(c_{ns}, \gamma_{ns}), p_g) \tag{3} \\
+    \end{align}
```
Comment (adamgayoso) [inline docs/user_guide/models/destvi.rst:87]: I would not include this part and just stick with the one below, ignoring how this is derived.

File: docs/user_guide/models/destvi.rst [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+:math:`p_g` is the rate parameter for the negative binomial distribution. So, we can model
+the cell type abundances :math:`\beta_{sc}` per spot rather than :math:`C(n)` and the
+corresponding cell types for each cell, we parametrize the sum from :math:`(3)` over cell types.
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+        x_{sg} &\sim \mathrm{NegativeBinomial}(l_s\alpha_g\sum_{c=1}^{C}\beta_{sc}f^g(c, \gamma_s^c), p_g) \tag{4} \\
+    \end{align}
```
Comment (adamgayoso) [inline docs/user_guide/models/destvi.rst:100]: I would include how gamma is generated (vampprior)

File: docs/user_guide/models/destvi.rst (hunk 2 above)
Comment (adamgayoso) [reply docs/user_guide/models/destvi.rst:162]: how are proportions computed? mathematically

**Review by adamgayoso — CHANGES_REQUESTED** [review]

File: docs/user_guide/models/destvi.rst (hunk 4 above)
Comment (romain-lopez) [reply docs/user_guide/models/destvi.rst:87]: yes this is exact! Let's do this!

File: docs/user_guide/models/destvi.rst (hunk 5 above)
Comment (romain-lopez) [reply docs/user_guide/models/destvi.rst:100]: yes!

**Review by romain-lopez — COMMENTED** [review]
Great! Thanks for writing this @jjhong922. I should take a pass later at the intro and incorporate some more ideas in there!

**Comment by adamgayoso** [comment]
Closing this as the history got messed up due to being based on a branch that got squashed. I cherrypicked the content in #1157

**Pull request closed without merging**